Makefile

```diff
@@ -260,20 +260,22 @@ bbr-image-kind: bbr-image-build ## Build the image and load it to kind cluster $
 
 ##@ Docs
 
-.PHONY: build-docs
-build-docs:
-	docker build --pull -t gaie/mkdocs hack/mkdocs/image
-	docker run --rm -v ${PWD}:/docs gaie/mkdocs build
-
-.PHONY: build-docs-netlify
-build-docs-netlify:
-	pip install -r hack/mkdocs/image/requirements.txt
-	python -m mkdocs build
+.PHONY: crd-ref-docs
```

Comment:
Removed unused doc targets and added new docs target to run the new make-docs script